cmd/cri-containerd/options/options.go

```diff
@@ -145,3 +150,22 @@ func (c *CRIContainerdOptions) InitFlags(fs *pflag.FlagSet) error {
 	// So I trigger twice parses, one for default value, one for commandline value.
 	return fs.Parse(commandline)
 }
+
+// GetDefaultTomlConfig print default toml config of cri-containerd.
+func (c *CRIContainerdOptions) GetDefaultTomlConfig() {
+	fs := pflag.NewFlagSet(os.Args[0]+"_getDefaultConfig", pflag.ExitOnError)
+
+	c.AddFlags(fs)
+
+	err := fs.Parse([]string{})
```

Comment:
nit:
```
if err := fs.Parse([]string{}); err != nil {
}
```
To save one line. And also for the following one.

Reply:
done